person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if @age || @parent_permission
+  end
```

Comment:
- [ ] (Optional) Great work, you don't have to write `return true` in Ruby because the language is structured to return the last line of code if a method is called. S you can write `@age || @parent_permission`. 👍